website/docs/commands/cli-config.html.markdown

```diff
@@ -60,27 +60,36 @@ The following settings can be set in the CLI configuration file:
   [plugin caching](/docs/configuration/providers.html#provider-plugin-cache)
   and specifies, as a string, the location of the plugin cache directory.
 
-- `credentials` — provides credentials for use with Terraform Enterprise's
-    [private module registry.](/docs/enterprise/registry/index.html) This is
-    only necessary when running Terraform on the command line; runs managed by
-    Terraform Enterprise can access private modules automatically.
+- `credentials` — provides credentials for use with Terraform Enterprise.
+    Terraform uses this when performing remote operations or state access with
+    the [remote backend](../backends/types/remote.html) and when accessing
+    Terraform Enterprise's [private module registry.](/docs/enterprise/registry/index.html)
+    Credentials are only needed when running Terraform locally on the command
+    line; runs managed by Terraform Enterprise automatically receive any
```

Comment:
I worry that this might be a little confusing in the case where the user runs `terraform plan` and the operation actually runs in Terraform Enterprise. In that case, it is both "locally on the command line" _and_ "runs managed by Terraform Enterprise" at the same time, and the credentials are needed because of the former.

Since we give users no way to directly customize the CLI config file in Terraform Enterprise, perhaps we can omit this statement altogether, and assume that _nothing_ on this page applies to runs happening inside Terraform Enterprise. (The truth is that there _is_ a CLI config in that case, but it's Terraform Enterprise itself that writes it rather than the end user, and so I think it's strange to mention it here.)

Reply:
Oh, that's a good point. I'll revise this sentence and then ping you again.

Reply:
@apparentlymart Yeah, after thinking about it I went ahead and just deleted that.